### ISSUE TITLE
Howto create ipns dnslink for ens app

### ISSUE THREAD

**Issue by chrisschaub** [ensdomains/ens-app#1627]
Since the ens app has removed support for dnslink style contentHashes like /ipns/somedomain.com , it would be great to have some documentation on the best way to create ipns hashes for dnslink. I can see in the code there are now checks to make sure ipns is crypto, not plain text.

I would like to see a working example of creating libp2p-key compatible contentHashes that can be used in the ens app. IPNS is a pain to setup and republish, but a compatible ipns hash should allow most ifps gateways to resolve the dnslink.

The most related github issue is here: https://github.com/ensdomains/ens-app/issues/849#issuecomment-660985090

Thank you for your consideration.

**Comment by rickstaa** [comment]
@chrisschaub I agree that some documentation on the recommended way of doing this in the ens-domain docs would be excellent. I followed https://docs-ipfs-tech.ipns.dweb.link/concepts/dnslink/#publish-content-path, and that method appears to work (see [this subdomain](ipfs.transcode.ninja)).
